rust/migration/src/m20220101_000001_create_blob_tables.rs

```diff
@@ -65,7 +65,7 @@ impl MigrationTrait for Migration {
                     .table(LocalBlobStore::Table)
                     .col(
                         ColumnDef::new(LocalBlobStore::Id)
-                            .integer()
+                            .uuid()
                             .not_null()
                             .primary_key(),
```

Comment:
Does this need the default call pointing to `gen_random_uuid`?

Reply:
Nice call out, in this case no but only for a sneaky reason. 

LocalBlobStore is a "dependent table" of the BlobStore (kind of like a subclass) and its ID is a foreign key to BlobStore's ID. That means the ID must always be manually set to and already existing BlobStore ID